releasenotes/notes/introduce-hamiltonian-gate-5c51bb576706cee9.yaml

```
@@ -1,7 +1,7 @@
 ---
 features:
   - |
-    New `HamitlotianGate` and `QuantumCircuit.hamltonian()` methods are
+    New `HamiltonianGate` and `QuantumCircuit.hamiltonian()` methods are
```

If you're correcting this can you make this sphinx references so these will be links to the documentation in the built output.

```suggestion
    New :class:``qiskit.extensionsHamiltonianGate` and :meth:`qiskit.circuit.QuantumCircuit.hamiltonian()` methods are
```

---

i think there's an extra ` after class

---

and a missing . after extensions